ray-curator/ray_curator/stages/text/io/writer/base.py

```diff
@@ -95,7 +108,7 @@ def process(self, task: DocumentBatch) -> FileGroupTask:
             data=[file_path],
             _metadata={
                 **task._metadata,
-                "output_dir": self.output_dir,
+                "output_dir": self.path,
```

Comment:
i think we can kill this metadata + format.. all that information is available in task.data anyway